src/utilities/useGetGlobalKdsAddress.ts

```diff
@@ -0,0 +1,3 @@
+export function useGetGlobalKdsAddress() {
+  return () => 'grpcs://<global-kds-address>:5685'
+}
```

Comment:
This feels like it should be part of our API client, plus from our conversation yesterday and from what I understand of the feature (if I understood correctly), I think this might belong there. Wdyt?

Reply:
Not really. It’s not clear whether this bit of information will come from the API or whether we’ll be able to determine it by means of some convention based on information already known in the environment.

Long story short, we simply don’t know yet, how we will get this.

Reply:
Ah ok, I'd misunderstood then, I thought we said it should be something callable (i.e. a function) as we are likely to get this from the backend. I think it also just feels 'network-y'/'http' like, but no bother, thanks for the extra detail.

Reply:
I mean it _is_ callable; the service factory returns a function and that is called (hence the slightly awkward name `useGetGlobalKdsAddress`).

Reply:
Yeah for sure, it is callable, and I'd understood that the reason for it needing to be callable was that we were gonna make a HTTP request with it in places to ask the backend for this info. 

> I thought we said it should be something callable (i.e. a function) as we are likely to get this from the backend

i.e. I wasn't saying it wasn't callable, I was saying we said it should be callable for the reason that we are likely to use it to get info from the backend, and that reason was the important point I was trying to make.

No matter, no action here.